azure-pipelines.yml

```diff
@@ -66,6 +74,10 @@ jobs:
         PYTHON: '3.12'
         CONDA_ENV: cienv
         WHEEL: 'yes'
+      py312_wheel:
```

Comment:
Is this suppose to be for 3.13?
it's failing azure: https://dev.azure.com/numba/numba/_build/results?buildId=17733&view=results